rx-android-extensions/src/main/java/com/appunite/rx/android/MoreFragmentActions.java

```diff
@@ -0,0 +1,30 @@
+package com.appunite.rx.android;
+
+import android.support.v4.app.Fragment;
+
+import javax.annotation.Nonnull;
+
+import rx.functions.Action1;
+
+public class MoreFragmentActions {
+
+    @Nonnull
+    public static Action1<? super Object> finishActivity(@Nonnull final Fragment fragment) {
```

Comment:
Don't like this because it can crash if there will not be activity associated and we should not silently call finishActivity on fragment. Activity should manage it's own lifecycle